pkg/kn/commands/service/service_list.go

```diff
@@ -52,6 +53,10 @@ func NewServiceListCommand(p *commands.KnParams) *cobra.Command {
 			if err != nil {
 				return err
 			}
+			//sort serviceList by name
+			sorter := NewServicesSorter(serviceList)
+			sort.Sort(sorter)
```

Comment:
How about sorting the list just before calling `PrintObj`?

Reply:
Good suggestion. Moved just before PrintObj. Thanks.